src/index.ts

```diff
@@ -1,2 +1,56 @@
-// eslint-disable-next-line no-console
-console.log("https://www.youtube.com/watch?v=dQw4w9WgXcQ")
+import path from "path"
+import fs from "fs"
+
+import { unknown } from "./unknown"
+import { optimize } from "./optimize"
+import { Debugger } from "./debugger"
+import { createIndexFiles } from "./createIndexFiles"
+import { components } from "./components"
+import { clean } from "./clean"
+import { build } from "./build"
+
+interface IndexFile {
+  path: string
+  extension: "tsx" | "svg"
+}
+
+export interface AssetsConfig {
+  rootPath: string
+  indexFiles?: IndexFile[]
+  replaceColors?: string[]
+  debug?: boolean
+}
+
+const loadConfiguration = (): AssetsConfig => {
+  const configPath = path.join(process.cwd(), "assets.config.json")
+
+  if (fs.existsSync(configPath)) {
+    return JSON.parse(fs.readFileSync(configPath).toString())
+  } else {
+    return {
+      rootPath: "./assets",
+      indexFiles: [],
+      replaceColors: [],
+      debug: false,
+    }
+  }
```

Comment:
Observation - the config file is not validated in any way.
We also don't check that the keys exist and do not provide fallbacks/defaults.

Reply:
added validation for the config file and corresponding error messages. Also, specified the defaults in the README. Thanks!